string action # `move to`, `move with` or `keep`
float32 setpoint # radians, radians/sec or newtons
---
bool is_success
